identifiers/country-us/census_autogenerated/us_congressional_districts.csv

```diff
@@ -447,3 +447,11 @@ ocd-division/country:us/state:ny/cd:29,New York's 29th congressional district (o
 ocd-division/country:us/state:oh/cd:17,Ohio's 17th congressional district (obsolete as of 2012),cd-3917,2013-01-03
 ocd-division/country:us/state:oh/cd:18,Ohio's 18th congressional district (obsolete as of 2012),cd-3918,2013-01-03
 ocd-division/country:us/state:pa/cd:19,Pennsylvania's 19th congressional district (obsolete as of 2012),cd-4219,2013-01-03
+ocd-division/country:us/state:il/cd:18,Illinois's 18th congressional district (obsolete as of 2022),cd-1718,2023-01-03

```

Comment:
I'm not a fan of the (obsolete as of X) being attached to the names since the validTo dates cover that information. However, I think it's fine to leave since it's consistent with the rest of the file.

Reply:
This is definitely a larger conversation, but I wonder if it might be prudent to add a pair of optionally fillable columns to indicate "effective on" and "obsolete on" dates.

Reply:
+1 To not being a fan of the "obsolete" formatting, but did want to maintain consistency